### PR TITLE
BF: visualise bundles mosaic when no streamlines

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: scripts/scil_visualize_bundles_mosaic.py [hunk 1]
```diff
@@ -262,7 +262,8 @@ def main():
                     colors = None
                 bundle_tractogram_file = nib.streamlines.load(bundle_file)
                 streamlines = bundle_tractogram_file.streamlines
-                bundle_actor = actor.line(streamlines, colors)
+                if len(streamlines) != 0:
```
Comment (GuillaumeTh) [inline scripts/scil_visualize_bundles_mosaic.py:265]: Only `if len(streamlines):`

File: scripts/scil_visualize_bundles_mosaic.py [hunk 2]
```diff
@@ -286,7 +287,8 @@ def main():
             ren.add(slice_actor)
 
             # Streamlines
-            ren.add(bundle_actor)
+            if len(streamlines) != 0:
```
Comment (GuillaumeTh) [inline scripts/scil_visualize_bundles_mosaic.py:290]: Only `if len(streamlines):`

**Review by GuillaumeTh — CHANGES_REQUESTED** [review]

**Review by GuillaumeTh — APPROVED** [review]

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Review by GuillaumeTh — APPROVED** [review]

**Pull request merged**